x/staking/keeper/validator.go

```diff
@@ -259,7 +258,7 @@ func (k Keeper) GetBondedValidatorsByPower(ctx sdk.Context) []types.Validator {
 func (k Keeper) ValidatorsPowerStoreIterator(ctx sdk.Context) (iterator sdk.Iterator) {
 	store := ctx.KVStore(k.storeKey)
 	iterator = sdk.KVStoreReversePrefixIterator(store, types.ValidatorsByPowerIndexKey)
```

Comment:
Why not just

```go
return sdk.KVStoreReversePrefixIterator(store, types.ValidatorsByPowerIndexKey)
```

Reply:
It seems that it is more common to use named return in current code base

Reply:
Not quite true. Named returns should only be done in clear and concise functions or with functions that have multiple return values of the same type to add context. Here there is no added benefit of named returns. It does not help readability at all.

Reply:
sounds good. I have updated the code based on the comment.